static/app/actionCreators/pageFilters.tsx

```diff
@@ -212,21 +216,25 @@ export function initializeUrlState({
    * IDs (project was deleted/moved to another org) can still exist in local storage or
    * shared links.
    */
-  function validateProjectId(projectId: number) {
+  function validateProjectId(projectId: number): boolean {
+    if (projectId === ALL_ACCESS_PROJECTS) {
```

Comment:
Do we need to validate that the user can use all projects here? Or is that validation done elsewhere?